web/src/main/kotlin/com/example/ajaxproject/service/interfaces/PrivateChatService.kt

```diff
@@ -1,17 +1,19 @@
 package com.example.ajaxproject.service.interfaces
 
-import com.example.ajaxproject.dto.request.RoomDTO
 import com.example.ajaxproject.dto.request.PrivateMessageDTO
+import com.example.ajaxproject.dto.request.RoomDTO
 import com.example.ajaxproject.model.PrivateChatMessage
 import com.example.ajaxproject.model.PrivateChatRoom
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
 
 interface PrivateChatService {
 
-    fun createPrivateRoom(senderId: String, recipientId: String): PrivateChatRoom
+    fun createPrivateRoom(senderId: String, recipientId: String): Mono<PrivateChatRoom>
 
-    fun getPrivateRoom(roomId: String): PrivateChatRoom?
+    fun getPrivateRoom(roomId: String): Mono<PrivateChatRoom>
 
-    fun sendPrivateMessage(privateMessageDTO: PrivateMessageDTO): PrivateChatMessage
+    fun sendPrivateMessage(privateMessageDTO: PrivateMessageDTO): Mono<PrivateChatMessage>
 
-    fun getAllPrivateMessages(roomDTO: RoomDTO): List<PrivateChatMessage>
+    fun getAllPrivateMessages(roomDTO: RoomDTO): Mono<List<PrivateChatMessage>>
```

Comment:
Can we return Flux instead of Mono list?

Reply:
It's just a question :)

Reply:
All is possible)